features/docs/writing_support_code/world_namespaces.feature

```diff
@@ -0,0 +1,186 @@
+Feature: World Namespaces
+
+  In order to better organise the code supporting my tests
+  As a cucumber user
+  I want to access the method added to a World using an explicit namespace
```

Comment:
It would be great to have some more explanatory text about this feature. I find the user story format a bit limiting for really good documentation.

Here's a good example: https://github.com/cucumber/cucumber-ruby/blob/master/features/docs/formatters/rerun_formatter.feature

If you're stuck for inspiration I can help out with the copy.